README.md

```diff
@@ -1,38 +1,22 @@
 Hy
 ==
 
-[![Build Status](https://img.shields.io/travis/hylang/hy/master.svg)](https://travis-ci.org/hylang/hy)
 [![Version](https://img.shields.io/pypi/v/hy.svg)](https://pypi.python.org/pypi/hy)
 
 <a href="https://xkcd.com/224/"><img title="We lost the documentation on quantum mechanics. You'll have to decode the regexes yourself." alt="XKCD #224" src="https://raw.github.com/hylang/shyte/18f6925e08684b0e1f52b2cc2c803989cd62cd91/imgs/xkcd.png"></a>
 
-Lisp and Python should love each other. Let's make it happen. [Try it](http://try-hy.appspot.com/).
+Lisp and Python should love each other. Let's make it happen.
 
-Hylarious Hacks
----------------
+Hy is a Lisp dialect that's embedded in Python. Since Hy transforms its Lisp
+code into Python abstract syntax tree (AST) objects, you have the whole
+beautiful world of Python at your fingertips, in Lisp form.
 
-* [Django + Lisp](https://github.com/paultag/djlisp/tree/master/djlisp)
-* [Python `sh` Fun](https://twitter.com/paultag/status/314925996442796032)
-* [Hy IRC Bot](https://github.com/hylang/hygdrop)
-* [miniKanren in Hy](https://github.com/algernon/adderall)
+To install the latest stable release of Hy, just use the command `pip3 install
```

Comment:
Side note: do we really want to be recommending pip3 without --user these days? It has a tendency to break...a lot. (Unless you're in a venv I guess, but if you *are* using venvs for apps, you might as well use pipx, right?)

Reply:
Oh dang, does it? I haven't noticed.

Reply:
Is `pip3 --user` more reliable, or is it just easier to clean up a mess in your home directory?

Reply:
More reliable; since `sudo pip3` installs to the same directory as distro package managers, it has a tendency to either overwrite the distro packages with wrong versions (broke apt for me back when I was on Ubuntu, not very fun to fix), or cause issues when you then try to install the distro version of a package that pip3 already installed. At this point [Debian defaults to --user](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=725848) (though that patch [also has its own issues](https://github.com/pypa/pip/issues/4222)).

Reply:
Okay, good to know. I'll add in the `--user`s and merge.